polaris-core/src/main/java/org/apache/polaris/core/PolarisCallContext.java

```diff
@@ -94,4 +94,13 @@ public RealmContext getRealmContext() {
   public PolarisCallContext getPolarisCallContext() {
     return this;
   }
+
+  @Override
+  public PolarisCallContext copy() {
+    // make a copy of the realm context
```

Comment:
nit: can we explain why we need a realm context copy here?

Reply:
I think I knew why, but a comment was supposed to explain why instead of what. It's easier to know what we do here by reading the code, but harder to know why we do that.

Reply:
The added text LGTM 👍